core/src/main/java/io/questdb/griffin/engine/table/LatestByValueFilteredRecordCursorFactory.java

```diff
@@ -54,11 +56,23 @@ public LatestByValueFilteredRecordCursorFactory(
         this.filter = filter;
     }
 
+    @Override
+    public String getBaseColumnName(int idx) {
```

Comment:
pls check what overrides this method - implementations are the same

Reply:
Fixed.